core/controllers/feedback.py

```diff
@@ -120,6 +120,23 @@ def get(self):
         })
 
 
+class FeedbackStatsHandler(base.BaseHandler):
+    """Returns Feedback stats for an exploration.
+        - Number of open threads
+        - Number of total threads
+    """
+    
+    def get(self, exploration_id):
+        feedback_thread_analytics = feedback_services.get_thread_analytics(
+            exploration_id)
+        self.values.update({
+            'num_open_threads': (
+                feedback_thread_analytics['num_open_threads']),
+            'num_total_threads': (
+                feedback_thread_analytics['num_total_threads']),
+        })
+        self.render_json(self.values)
+
```

Comment:
add one more newline here (we separate top-level classes/functions with two newlines)